internal/service/memory/helper_service.go

```diff
@@ -97,7 +97,7 @@ func CreateMapUser(shu *ShortURLAttr) (map[string]bool, error) {
 }
 
 // DelURLs делает отметку об удалении коротких URL-ы определенного пользователя
-func (s *Service) DelURLs(cookieValue string, buf bytes.Buffer) (bool, error) {
+func (s *Service) DelURLs(ctx context.Context, cookieValue string, buf bytes.Buffer) (bool, error) {
```

Comment:
```suggestion
func (s *Service) DelURLs(userIDEncoded string, buf bytes.Buffer) (bool, error) {
```
А по хорошему, сюда должен прилетать сразу распакованный User. Контекст от запроса в данную функцию пробрасывать нельзя, я выше писал, почему.